src/applications/personalization/profile/components/Profile.jsx

```diff
@@ -294,20 +294,25 @@ const mapStateToProps = state => {
   const showLoader =
     !hasLoadedAllData || (!isLOA3 && !isLOA1 && currentlyLoggedIn);
 
+  const shouldShowDirectDeposit = () => {
+    // if they are explicitly blocked from DD4CNP, do not show it
+    if (isCNPDirectDepositBlocked) return false;
```

Comment:
Rookie question - Is it possible for users to be blocked for CNP but not for DD for EDU? 

And I agree this is more readable! 💯

Reply:
Excellent question! The DD4EDU endpoint is minimal and won't tell us if someone is flagged as incompetent, having a fiduciary, etc. So I _hope_ that if someone only gets DD4EDU, the endpoint will simply treat them as not having DD4EDU set up if they should be blocked from the feature for some reason. Otherwise it's entirely possible someone who get DD4EDU _and_ has a fiduciary will have access to their DD4EDU info in the Profile.

Given that we show users a helpful message if they are explicitly blocked from DD due to having a fiduciary, etc. I think this is the correct handling. If someone is flagged as being blocked from DD4CNP, we will also be blocking them from accessing their DD4EDU info. We are showing a helpful message in this case, so if there is some crazy edge case and this person actually should have access to their DD4EDU info despite being explicitly blocked from DD4CNP, they can call someone and get help.

Hopefully that's clear enough. This is all a little confusing. And also not satisfying to know for sure what the correct approach is.

Gonna tag @Samara-Strauss and @mshea0606 so they can read my comment here, see if they can make sense of it, and decided if this is the correct handling!

Reply:
I think it should be fine as is — DD for edu is a lot smaller and a lot less of a concern than comp & pen payments. It's basically for books and supplies (actual tuition payments go directly to the schools). But, for the sake of due diligence, I'll reach out to Melissa Rebstock to confirm that there aren't fiduciary issues here. I'll let you know what she says.